src/seedu/addressbook/parser/Parser.java

```diff
@@ -226,5 +228,119 @@ private Command prepareFind(String args) {
         return new FindCommand(keywordSet);
     }
 
+    /**
+     * Feedback a suggested command to the user.
+     *
+     * @param incorrectCommandWord the incorrect command word
+     * @return the suggested command
+     */
+    private IncorrectCommand suggestCommand(String incorrectCommandWord) {
+        String suggestedCommandWord = findMostProbableCommand(incorrectCommandWord);
+        switch (suggestedCommandWord) {
+
```

Comment:
These looks like copy and paste. There's a better way to refactor this (hint: You may need to tweak your `findMostProbableCommand` a bit